portfolio/src/main/webapp/script.js

```diff
@@ -29,7 +29,14 @@ function addMusic() {
 
 async function showMessage() {
 const responseFromServer = await fetch('/hello');
+const textFromResponse = await responseFromServer.json;
+let pick = Math.floor(Math.random() * textFromResponse.length);
+const helloContainer = document.getElementById("hello-container");
+helloContainer.innerText = textFromResponse[pick];
+
+/*
```

Comment:
Better to remove commented out code. The benefit of keeping the world in `git` is that you can always inspect the old version of the file.